lib/process-services/src/lib/services/apps-process.service.ts

```diff
@@ -18,8 +18,7 @@
 import { Injectable } from '@angular/core';
```

Comment:
Similar comment as above- I'd avoid stacking up all of the services in one `services` folder but rather organize them by domain/area on what they actually do. If possible of course :) If not, let me know why :)

Reply:
In this specific case, this service was moved to process services but kept in a general services folder because it is used across multiple modules across the package.